package.json

```diff
@@ -34,37 +34,37 @@
   "peerDependencies": {
     "react": "^17.0.0 || ^16.9.0",
```

Comment:
To fix the issue you described in #62 you would have to add `^18.0.0` here.

Reply:
Aha!